File: timetable-front/src/main/webapp/partials/js/br/edu/ifma/csp/timetable/component/LookupBox.js
br.edu.ifma.csp.timetable.component.LookupBox = zk.$extends(zul.Widget, {
	$define: {
		msg: function(v) {
			var n;
			
			if (v && (n = this.$n))
				n.innerHTML = v;
		},
		description: null
	},
	
	getZclass: function() {
		
		var zcls = this._zclass;
		return zcls ? zcls : 'z-lookupbox';
	}
});

File: timetable-front/src/main/java/br/edu/ifma/csp/timetable/component/LookupBox.java
package br.edu.ifma.csp.timetable.component;

import org.zkoss.zk.ui.IdSpace;
import org.zkoss.zul.Bandbox;

/**
 * @author inalberth
 *
 */
public class LookupBox extends Bandbox implements IdSpace {

	private static final long serialVersionUID = -100982316182135654L;
	
	@Override
	public String getZclass() {
		return _zclass == null ? "z-lookupbox" : _zclass;
	}
	
}
